typegen/src/utils.rs

```diff
@@ -263,7 +285,7 @@ mod generics_list {
     struct GenericsListInner {
         previous: Option<GenericsList>,
         start_idx: usize,
-        generics_by_id: Vec<u32>,
+        generics_by_id: Vec<(u32, String)>,
```

Comment:
Does this store the type id and type name? Or why was it changed?

Reply:
Removed usize index from there

Reply:
I was comparing to the old code I missed your change that you and James discussed :)